src/lib.rs

```diff
@@ -370,7 +370,7 @@ impl Mnemonic {
 	/// word lists. In the extremely unlikely case that a word list can be
 	/// interpreted in multiple languages, an [Error::AmbiguousLanguages] is
 	/// returned, containing the possible languages.
-	fn language_of<S: AsRef<str>>(mnemonic: S) -> Result<Language, Error> {
+	pub fn language_of<S: AsRef<str>>(mnemonic: S) -> Result<Language, Error> {
```

Comment:
I think your observation from #10 is still true that you can use the narrower error type here.

Reply:
No because here S is a string so it can be an invalid mnemonic. This is the static method used for parsing.